### DISCUSSION TITLE
Redis requirement clarification

### DISCUSSION THREAD

**Discussion by auanasgheps** [jonaswinkler/paperless-ng#607] [General]
Hello,
The Docker Compose for this application is using Redis 6.0

I have other containers relying on Redis, but they are fine with the latest release tag, which currently is Redis 6.2.

Is there any technical reason to stick with 6.0 tag?

Thanks

**Comment by jonaswinkler** [comment]
Probably not, it might just work.

It just that I don't exactly follow updates and changes to redis, and by specifying an exact version here prevents updates to redis from possibly breaking paperless.

**Reply by auanasgheps** [reply]
OK thanks!
